engine/univSubst.mli

```diff
@@ -41,3 +41,12 @@ val nf_evars_and_universes_opt_subst : (existential -> constr option) ->
 val subst_univs_universe : (Level.t -> Universe.t) -> Universe.t -> Universe.t
 
 val pr_universe_subst : universe_subst -> Pp.t
+
+val enforce_eq : Universe.t constraint_function
```

Comment:
These functions don't really belong in univsubst
Maybe univproblem?

Reply:
That's the lower place where we start using them. Given there is a `enforce_univ_constraint` right below it feels natural to put them there. If you want to move them somewhere else again it's still possible to do after this PR. AFAIK no plugins from the CI are relying on these low-level functions anymore.